exercises/01.e2e/01.problem.init/other/build-server.ts

```diff
@@ -3,7 +3,8 @@ import path from 'path'
 import glob from 'glob'
 import pkg from '../package.json'
 
-const here = (...s: Array<string>) => path.join(__dirname, ...s)
+const here = (...s: Array<string>) =>
+	path.join(__dirname, ...s).replace(/\\/g, '/')
```

Comment:
This affects more than just the glob. Could you update this PR to only change the glob instead?

Reply:
OK